sdk/openai/Azure.AI.OpenAI/tests/AssistantTests.cs

```diff
@@ -122,7 +122,7 @@ public async Task SettingResponseFormatWorks()
         Assert.That(assistant.ResponseFormat, Is.EqualTo(AssistantResponseFormat.Text));
         AssistantThread thread = await client.CreateThreadAsync();
         Validate(thread);
-        ThreadMessage message = await client.CreateMessageAsync(thread, ["Write some JSON for me!"]);
+        ThreadMessage message = await client.CreateMessageAsync(thread.Id, MessageRole.User, ["Write some JSON for me!"]);
```

Comment:
The changes are not caused by the PR. 
`Azure.AI.OpenAI` SDK has no code change. But it depended SDK `OpenAI` had released a new version 2.0.0-beta.6 which has breaking change. So updated the `AssistentTest.cs` to comply with the new 2.0.0-beta.6 OpenAI library.

Reply:
Instead of updating the code here, can we use the fixed version of the dependency library?

Reply:
I think `Azure.AI.OpenAI` SDK will release from azure-sdk-for-net repo, and it may be need to adopt the new version in `openai/openai-dotnet` to include the new apis.
I am confirming with @m-nash about this: whether we will pin the `openAI` version in `Azure.AI.OpenAI` SDK.